src/main/java/com/softserve/demo/controller/RegisterController.java

```diff
@@ -19,6 +20,15 @@ public RegisterController(final RegisterService registerService) {
         this.registerService = registerService;
     }
 
+    @GetMapping("verify/{activationCode}")
+    public String verifyUserEmail(@PathVariable String activationCode) {
+        if (registerService.verifyUser(activationCode)) {
+            return "Successfully verified!";
+        } else {
```

Comment:
We do not need else here. We already have return in first if statement. After that we can simply throw exception

Reply:
Maybe this else block is really unnecessary here. But I think that it provides better readability, isn't it? With this block defined explicitly I can see two distinct cases.